### PR TITLE
Normalize project notes author tracking

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/9644

Here's another patch I want to get squared away ahead of the access migration.

## Testing
**URL to test:** Local only 

**Steps to test:**

To test the migrations:

1. Follow all the steps to [load production](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-MIQvl_rKnZ_-wHRdp4J/dev-guides/how-tos/how-to-load-production-data-into-a-local-instance) data into your local instance.
2. Connect to the prod read replica, and observe that `added_by_user_text` and  `added_by_user_name` do not match when you run this query:

```sql
select mpn.project_note_id, mpn.date_created, mpn.added_by as added_by_user_text, mu.first_name || ' ' || mu.last_name as added_by_user_name
from moped_proj_notes mpn
left join moped_users mu on mpn.added_by_user_id = mu.user_id
where mpn.date_created < '2022-07-15';
```

3. Run the same query against your local instance. in all cases, `added_by_user_text` should match `added_by_user_name`. The migrations were successful!

You will not be able to test the Moped editor after loading in production data, because our local user (id = 1) does not exist in the production database that you replicated locally. You'll receive an error if you try to create a new comment or status update.

To test the editor:

1. Stop the hasura cluster and start it again.

2. Start the Moped Editor. Inspect comments and status update to verify the comment author name is rendering correctly. 

3. Create and edit comments as well. You should do this in these places:

- Project comments tab
- Project summary status update
- Status update column from the Dashboard page

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/queries/project.js [hunk 1]
```diff
@@ -696,28 +698,6 @@ export const PROJECT_SUMMARY_STATUS_UPDATE_INSERT = gql`
...
-        project_note: $project_note
-        added_by: $added_by
-        project_note_type: 2
-      }
-    ) {
-      affected_rows
-    }
-  }
-`;
-
```
Comment (johnclary) [inline moped-editor/src/queries/project.js:720]: we weren't using this mutation anymore

Comment (chiaberry) [reply moped-editor/src/queries/project.js:720]: 🧹

**Review by chiaberry — APPROVED** [review]
Followed all the steps to test and it worked!

File: moped-editor/src/queries/comments.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ export const COMMENTS_QUERY = gql`
       where: $projectNoteConditions
       order_by: { date_created: desc }
     ) {
-      added_by
-      added_by_user_id
+      moped_user {
```
Comment (mddilley) [inline moped-editor/src/queries/comments.js:9]: 🙌

**Review by mddilley — APPROVED** [review]
Followed the test steps, and all works beautifully! Seeing the payoff in the code updates of these normalization PRs never gets old. Thanks for updating! 🙏 🚢

**Review by frankhereford — APPROVED** [review]
Wholehearted agreement with Mike that these normalizations do never get old! Thank you John!

I followed your instructions, and everything worked just like one would expect. The code and migrations both look good to me as well. This is right on the money - 💯.

**Comment by johnclary** [comment]
Thanks for your reviews!

**Pull request merged**